focus_converter_base/focus_converter/conversion_configs/oci/1_pricing_quantity_S001.yaml

```diff
@@ -0,0 +1,4 @@
+plan_name: convert usage/billedQuantity to PricingQuantity
+conversion_type: rename_column
+column: usage/billedQuantity
+focus_column: PricingQuantity
```

Comment:
According to FOCUS, the following should apply: `
List/Effective/Billed UnitPrice * PricingQuantity = List/Effective/Billed Cost`

Considering that, when covered with Universal Credits, you are billed for `billedQuantityOverage` instead `billedQuantity` we might have to split some records into two?!

I propose initiating a **Discussion topic** in the **FOCUS project**.

_Note: Similar concerns apply to UsageQuantity mapping._